packages/frontend/component/src/ui/icon-picker/icon-picker.stories.tsx

```
@@ -0,0 +1,20 @@
+import { IconPicker } from './icon-picker';
+
+export default {
+  title: 'UI/IconPicker',
+  component: IconPicker,
+};
+
+export const Default = () => {
+  return (
+    <>
+      <div>Selected: // TODO</div>
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)